src/style.css

```
@@ -1 +1,15 @@
-/* styles go here */
+.input {
```

you might not set class `input` where you don't need margin-top

**or**

```suggestion
.input:not(:first-child) {
```